src/core/opamFilename.ml

```diff
@@ -109,8 +109,10 @@ let exec dirname ?env ?name ?metadata ?keep_going cmds =
     (fun () -> OpamSystem.commands ?env ?name ?metadata ?keep_going cmds)
 
 let move_dir ~src ~dst =
-  OpamSystem.command ~verbose:(OpamSystem.verbose_for_base_commands ())
-    [ "mv"; Dir.to_string src; Dir.to_string dst ]
+  (* XXX Windows - destination must be on the same drive or this fails *)
```

Comment:
Isn’t that the same behaviour as on Unix? `Sys.rename` (morrally) uses `Unix.rename` afaik